interface/src/app/app.js

```diff
@@ -19,15 +20,37 @@ angular.module( 'adage', [
   $resourceProvider.defaults.stripTrailingSlashes = false;
 }])
 
-.run( function run () {
+.run( function run ( $state, $location ) {
+
+    // Will only redirect if there is no state AND the url is not a
+    // route for tribe_client
+    String.prototype.startsWith = function(prefix) {
+        return this.slice(0, prefix.length) == prefix;
+    };
+
+    if ($state['current']['name'] === '') {
+      if (window.location.pathname.startsWith('/tribe_client')) {
```

Comment:
@cgreene - I agree, I think that would probably be the most pragmatic way of doing it so that we could then just use it in different webservers. The catch then will be that the UI of that webserver will have to use Angular to be able to use that app, but it is definitely much better than having not having a re-installable app for the front end.